### DISCUSSION TITLE
setting up dev env

### DISCUSSION THREAD

**Discussion by akshitsaini111** [apache/airflow#31225] [General]
How to setup dev env

**Comment by potiuk** [comment]
https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst

**Comment by akshitsaini111** [comment]
setting up the dev env is quite stiff.

**Reply by potiuk** [reply]
> setting up the dev env is quite stiff.

Any way to make it simpler, and easier is most welcome as your first PR. This has been contributed by many people who worked hard to describe it in the way that you have screenshots and guidelines. Usually in their free time. 

If you think you can figure out the way to make it less stiff (whatever this means) and be able to develop airflow - this is a great chance for you to give back to the community and make it "less stiff". All contributions are welcome.
